librz/core/cmd_descs/cmd_search.yaml

```diff
@@ -3,8 +3,174 @@
+    args:
+      - name: opcodes
+        type: RZ_CMD_ARG_TYPE_STRING
+
+  - name: "/c"
+    summary: Cryptographic material search
+    cname: cmd_cryptographic_material_search
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    default_mode: RZ_OUTPUT_MODE_STANDARD
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_QUIET
+      - RZ_OUTPUT_MODE_TABLE
+    args: []
+
+  - name: "/e"
+    summary: Raw regular expression search.
+    cname: cmd_regex_raw_search
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    default_mode: RZ_OUTPUT_MODE_STANDARD
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_QUIET
+      - RZ_OUTPUT_MODE_TABLE
+    args:
+      - name: regex
+        type: RZ_CMD_ARG_TYPE_STRING
+
+  - name: "/g"
+    summary: Search for all graph paths A to B (/gg follow jumps, see search.count and analysis.depth).
+    subcommands:
+    - name: "/g"
+      summary: Search for all graph paths A to B (does not follow jumps).
+      cname: cmd_graph_path_search
+      type: RZ_CMD_DESC_TYPE_ARGV_STATE
+      default_mode: RZ_OUTPUT_MODE_STANDARD
+      modes:
+        - RZ_OUTPUT_MODE_STANDARD
+        - RZ_OUTPUT_MODE_JSON
+        - RZ_OUTPUT_MODE_QUIET
+        - RZ_OUTPUT_MODE_TABLE
+      args:
+        - name: from
+          type: RZ_CMD_ARG_TYPE_RZNUM
+        - name: to
+          type: RZ_CMD_ARG_TYPE_RZNUM
+    - name: "/gg"
+      summary: Search for all graph paths A to B (follow jumps, see `search.count` and `analysis.depth`).
+      cname: cmd_graph_path_follow_jumps_search
+      type: RZ_CMD_DESC_TYPE_ARGV_STATE
+      default_mode: RZ_OUTPUT_MODE_STANDARD
+      modes:
+        - RZ_OUTPUT_MODE_STANDARD
+        - RZ_OUTPUT_MODE_JSON
+        - RZ_OUTPUT_MODE_QUIET
+        - RZ_OUTPUT_MODE_TABLE
+      args:
+        - name: from
+          type: RZ_CMD_ARG_TYPE_RZNUM
+        - name: to
+          type: RZ_CMD_ARG_TYPE_RZNUM
+
+  - name: "/h"
+    summary: Search for blocks that has the same given hash (see also command `ph`).
+    cname: cmd_hash_block_search
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    default_mode: RZ_OUTPUT_MODE_STANDARD
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_QUIET
+      - RZ_OUTPUT_MODE_TABLE
+    args:
+      - name: algo
+        type: RZ_CMD_ARG_TYPE_STRING
+      - name: hash
+        type: RZ_CMD_ARG_TYPE_STRING
+    details:
+      - name: Usage example
+        entries:
+          - text: "MD5 hash search within blocks of 512 bytes."
+            comment: "/h md5 0bc8f8c426b74ffaedac8330a7464014 @! 512"
+
+  - name: "/m"
+    summary: Magic constants search.
+    cname: cmd_magic_const_search
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    default_mode: RZ_OUTPUT_MODE_STANDARD
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_QUIET
+      - RZ_OUTPUT_MODE_TABLE
+    args: []
+
+  - name: "/p"
```

Comment:
For every search subcommand we need more detailed description and few examples.

Reply:
i think `/p` will be removed since it was linked to a function which nobody knew what was doing nor the original help explained.